pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -124,6 +124,58 @@ def execute_startup_script(file_path, verbose):
                 err=True)
             raise click.Abort()
 
+def _resolve_mock_server(mock_server):
+    """
+    Validate and resolve the mock_server parameter.
+    Validates that all files defined in the server exist and that the
+    types are valid.
+
+    TODO: Confirm that it is necessary to join the basename to cwd in the
+    case only basename is provided for all OSs.
+
+    Parameters:
+
+      mock_server (list of _resolve_mock_server):
+        Strings defining all of the files that compromise the mock_server
+        definiton.
+
+    Returns: list of resolved mock server definition components derived
+       from the input parameter mock_server.
+
+    Raises:
+      click.ClickException for any errors.
+    """
+    resolved_mock_server = []
+    assert isinstance(mock_server, tuple)
```

Comment:
Description says list, but code checks for tuple. Make consistent. Why not accept Sequence?

Reply:
FIXED